tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -61,7 +61,7 @@ <h2>Settings</h2>
     <paper-header-panel>
       <paper-toolbar id="toolbar" slot="header" class="header">
         <div id="toolbar-content" slot="top">
-          <div class="toolbar-title">[[brand]]</div>
+          <div class="toolbar-title"><a href="/">[[brand]]</a></div>
```

Comment:
This will not work AFAIK. Can you please test this with path_prefix flag?

Reply:
Confirmed bad behavior with path_prefix. 

Looks like next simplest option is to add a parameter to the tf-tensorboard component, and pass in the link target.  Only render the link if the link target is non-null/empty. -- I don't see an option that is likely to do the right thing in the presence of path_prefix across the mutliple deployment types.